From 4366920bb2a97c10c49c5e6d035c0c82629b9f0a Mon Sep 17 00:00:00 2001
From: Cristian Stoica <cristian.stoica@nxp.com>
Date: Mon, 8 Feb 2016 17:15:25 +0200
Subject: [PATCH 39/48] cryptodev: explicitly discard const qualifier

The const qualifier is discarded by the assignment as a result of how
the variables are defined. This patch drops the const qualifier
explicitly to avoid build errors.

Signed-off-by: Cristian Stoica <cristian.stoica@nxp.com>
---
 crypto/engine/eng_cryptodev.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crypto/engine/eng_cryptodev.c b/crypto/engine/eng_cryptodev.c
index 4296704..f8619b0 100644
--- a/crypto/engine/eng_cryptodev.c
+++ b/crypto/engine/eng_cryptodev.c
@@ -1593,7 +1593,7 @@ static int cryptodev_digest_update(EVP_MD_CTX *ctx, const void *data,
      * cryptodev calls and accumulating small amounts of data
      */
     if (ctx->flags & EVP_MD_CTX_FLAG_ONESHOT) {
-        state->mac_data = data;
+        state->mac_data = (void *)data;
         state->mac_len = count;
     } else {
         state->mac_data =
-- 
2.7.3

